actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -9,7 +9,7 @@ module CollectionCaching # :nodoc:
     included do
       # Fallback cache store if Action View is used without Rails.
       # Otherwise overridden in Railtie to use Rails.cache.
-      mattr_accessor :collection_cache, default: ActiveSupport::Cache::MemoryStore.new
+      mattr_accessor :collection_cache, default: ActiveSupport::Cache::NullStore.new
```

Comment:
This definitely doesn't sound backportable

Reply:
It's not strictly necessary, just seemed weird to me. I'll revert it.